tiledb/libtiledb.pyx

```diff
@@ -1534,6 +1534,7 @@ cdef class Attr(object):
         if isinstance(dtype, str) and dtype == "ascii":
             tiledb_dtype = TILEDB_STRING_ASCII
             ncells = TILEDB_VAR_NUM
+            var = True
```

Comment:
Would still like for `tiledb.Attr(dtype="ascii", var=False)` to error out. So only set to True if var is None.

```
diff --git a/tiledb/libtiledb.pyx b/tiledb/libtiledb.pyx
index b223345..327f69f 100644
--- a/tiledb/libtiledb.pyx
+++ b/tiledb/libtiledb.pyx
@@ -1534,7 +1534,8 @@ cdef class Attr(object):
         if isinstance(dtype, str) and dtype == "ascii":
             tiledb_dtype = TILEDB_STRING_ASCII
             ncells = TILEDB_VAR_NUM
-            var = True
+            if var is None:
+                var = True
         else:
             _dtype = np.dtype(dtype)
             tiledb_dtype, ncells = array_type_ncells(_dtype)
```